src/mumble/SettingsMacros.h

```diff
@@ -21,6 +21,7 @@
 #define AUDIO_SETTINGS                                                                      \
 	PROCESS(audio, MUTE_KEY, bMute)                                                         \
 	PROCESS(audio, DEAF_KEY, bDeaf)                                                         \
+	PROCESS(audio, AUTO_UNMUTE_KEY, autoUnmute)                                             \
```

Comment:
```suggestion
	PROCESS(audio, AUTO_UNMUTE_KEY, autoUnmute)                                             \
	PROCESS(audio, MUTE_KEY, bMute)                                                         \
	PROCESS(audio, DEAF_KEY, bDeaf)                                                         \
```

Reply:
This is just the order. I did not mess up the whitespace, right?

Reply:
Just the order.